### DISCUSSION TITLE
Mitosis

### DISCUSSION THREAD

**Discussion by amundlindstol** [digdir/designsystemet#2090] [Feature requests]
Hei! Godt jobba🥇
Har dere vurdert å ta i bruk [mitosis](https://github.com/BuilderIO/mitosis)? 😁
Hilsen svelte

**Comment by mimarz** [comment]
Hei! 

Bra forslag :) 

Vi følger med på både mitosis, stenciljs og andre "write-once-compile-everywhere" rammeverk.

Av erfaring så funker det fint for enkelte komponenter, men kan fort bli kluss eller begrensende med mer avansert komponenter. Du gifter deg med slike rammeverk, så enn må og vurdere ting som hvor pålitelig rammeverket, vedlikeholdskost osv. 

Støtte for andre platformer er planen først å få på plass et rent css-rammeverk og så ta en liten runde med research på ulike slike rammeverk og ikke minst web-components! #1044  

Ideelt så ser jeg for meg at vi løser støtte på tvers av frontend bibliotek og rammeverk med web-components og ren css, men ja, det er ikke uten sine begrensninger eller utfordringer :)
